gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/CompoundStatementResourceExtractors.java

```diff
@@ -120,15 +120,25 @@ public static Stream<RCMRMT030101UK04RequestStatement> extractAllRequestStatemen
         );
     }
 
-    public static Stream<RCMRMT030101UK04NarrativeStatement> extractAllNarrativeStatements(RCMRMT030101UK04Component4 component4) {
-        return Stream.concat(
-            Stream.of(component4.getNarrativeStatement()),
-            component4.hasCompoundStatement() ? CompoundStatementUtil.extractResourcesFromCompound(component4.getCompoundStatement(),
-                    RCMRMT030101UK04Component02::hasNarrativeStatement, RCMRMT030101UK04Component02::getNarrativeStatement)
-                .stream()
-                .map(RCMRMT030101UK04NarrativeStatement.class::cast)
-                : Stream.empty()
-        );
+    public static Stream<RCMRMT030101UK04NarrativeStatement> extractAllNarrativeStatements(
```

Comment:
Rename refactor to `extractAllNarrativeStatementsExcludingBloodPressure`.

Reply:
Yup, that was on my list, have corrected this now